### DISCUSSION TITLE
What is  the meaning of ACCG and CROG in Json example?

### DISCUSSION THREAD

**Discussion by musukvl** [b3b00/csly#407] [Q&A] [answered]
Could shade light on ACCG and CROG abbriviations, what do they mean? 
https://github.com/b3b00/csly/blob/86697b24b10d752c9d1e74d6b92aa7c6e8b38c47/src/samples/jsonparser/JsonTokenGeneric.cs#L13

**Comment by b3b00** [comment, marked as answer]
@musukvl ,

This is french . it stands for : 
- ACCG : ACColade Gauche : opening curly brace
- CROG : CROchet Gauche : opening suare bracket.

It could have be anything else as long as you keep consistent in production rules.
